go-controller/pkg/ovnwebhook/user.go

```diff
@@ -7,6 +7,17 @@ import (
 	authenticationv1 "k8s.io/api/authentication/v1"
 )
 
+func checkNodeIdentity(csrAcceptanceConditions []csrapprover.CSRAcceptanceCondition, user authenticationv1.UserInfo) (bool, *csrapprover.CSRAcceptanceCondition, string) {
```

Comment:
If `ovnkubeNodeIdentity` is no longer used please remove it.
Additionally please add a doc string.

Reply:
I think the node webhook should be handled pretty much the same as the pod one, where we should also allow to add the additional annotation checks.
I am ok to leave that out for now considering we are under a time constraint.